.changeset/chilled-kings-attend.md

```diff
@@ -0,0 +1,5 @@
+---
+"observability": patch
+---
+
+Provision RoleTemplates for RBAC
```

Comment:
it does not look like a patch but more like extending the existing API so it must be a minor maybe?

Reply:
maybe - I was just trying to get something going here.

Reply:
changed it to minor - let's see what the semantic thing thinks of that